ext/AbstractDifferentiationEnzymeExt.jl

```diff
@@ -0,0 +1,60 @@
+module AbstractDifferentiationEnzymeExt
+
+if isdefined(Base, :get_extension)
+    import AbstractDifferentiation as AD
+    using Enzyme: Enzyme
+else
+    import ..AbstractDifferentiation as AD
+    using ..Enzyme: Enzyme
+end
+
+AD.@primitive function jacobian(b::AD.EnzymeForwardBackend, f, x)
```

Comment:
No need for `@primitve` (see #95):
```suggestion
function AD.jacobian(b::AD.EnzymeForwardBackend, f, x)
```

Reply:
redefined in terms of pushforward and pullback